chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -825,13 +822,13 @@ public synchronized void loadChunks(TaskTracker progress, World world,
           int wz = cz + wz0;
           for (int cx = 0; cx < 16; ++cx) {
             int wx = cx + wx0;
-            int biomeId = 0xFF & biomes[Chunk.chunkXZIndex(cx, cz)];
+            int biomeId = 0xFF & chunkData.getBiomeAt(cx, 0, cz);
```

Comment:
While you're at it, you might as well add a `// TODO add vertical biomes support (1.15+)`